journeys/0xCUBE.md

```diff
@@ -0,0 +1,51 @@
+<p align="center">
+  <img src="images/Small-Cube.png" />
```

Comment:
image not working

![image](https://user-images.githubusercontent.com/84712013/181920475-2e3596ac-f6ae-4fc3-b22f-e660a9f8b505.png)